src/main/java/seedu/address/logic/commands/DeleteTimeCommand.java

```diff
@@ -98,6 +102,12 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
             }
         }
 
+        // Case 2: None of the specified free time(s) match any of the person's free time
+        if (freeTimeTags.size() == updatedFreeTimeTags.size()) {
+            throw new CommandException(Messages.MESSAGE_NO_MATCHING_FREE_TIME);
+        }
```

Comment:
`freeTimeTags` is the original free time tags, and `updatedFreeTimeTags` is the new free time tags.

If they have the same size after the `deleteTime` command, it means nothing is deleted, and hence implies no matching.